proposed/http-client/http-client-meta.md

```diff
@@ -39,10 +39,8 @@ specify the default behaviours.
 ### Default behavior
 
 The intention of this PSR is ensure library developers that all HTTP clients have the same 
```

Comment:
i think "... PSR is ensure library developers" this is not correct english.

How about we do it like this: (also includes a proposal for the Liskov substitution):

 "The intention of this PSR is to provide library developers with HTTP clients that have a well defined behaviour. A library should be able to use any compliant client without special code to handle client implementation details (Liskov substitution principle)."